src/web/components/GuardianLines.stories.tsx

```diff
@@ -58,26 +54,33 @@ export const eightLines = () => {
                 <GuardianLines count={8} />
             </Section>
             <Section showTopBorder={false}>
-                <Flex>
-                    <LeftColumn>
-                        <></>
-                    </LeftColumn>
-                    <ArticleContainer>
-                        <ArticleHeadline
-                            headlineString="Headline text"
-                            webPublicationDate=""
-                            tags={[]}
-                            designType="Article"
-                            pillar="news"
-                        />
-                    </ArticleContainer>
-                </Flex>
+                <ArticleContainer>
+                    <ArticleHeadline
+                        headlineString="Headline text"
+                        webPublicationDate=""
+                        tags={[]}
+                        designType="Article"
+                        pillar="news"
+                    />
+                </ArticleContainer>
             </Section>
         </>
     );
 };
 eightLines.story = { name: 'with eight lines' };
 
+// @ts-ignore
```

Comment:
Ditto on ts-ignore. Stories should be test cases for how the component works so if typescript is failing that needs to be fixed.